### PR TITLE
console: remove ERRINJ_STDIN_ISATTY injection

### LINKED ISSUE #7554: Remove `ERRINJ_STDIN_ISATTY`
#5064 is fixed, now we can remove `ERRINJ_STDIN_ISATTY` injection as excessive.

### PR DESCRIPTION
**Pull request by Lord-KA**
As the underlying problem behind this injection is fixed in #7357 it can
be removed and `-i` flag could be used as initially intended.

Closes #7554
Requires #7357
NO_DOC=refactoring
NO_CHANGELOG=refactoring

### CODE REVIEW

File: test/app-tap/gh-5040-inter-mode-isatty-via-errinj.test.lua [hunk 1]
```diff
@@ -7,8 +7,7 @@ local fio = require('fio')
```
Comment (Totktonada) [inline test/app-tap/gh-5040-inter-mode-isatty-via-errinj.test.lua:7]: How about `test/app-tap/gh-2717-no-quit-sigint.test.lua`?

**Comment by Totktonada** [comment]
Please, don't set `full-ci` label before a review if your change is not about a specific platform, requires integration testing, etc. The minimal CI usually sufficient to check a pull request. The successful `full-ci` is a check-in criteria and it is set by a maintainer before a push to `master`. If we'll run `full-ci` for each push, it'll consume much machine resources and developers will wait for their checks longer.

File: test/app-tap/gh-5040-inter-mode-isatty-via-errinj.test.lua (hunk 1 above)
Comment (Lord-KA) [reply test/app-tap/gh-5040-inter-mode-isatty-via-errinj.test.lua:7]: Thx, missed that.

**Review by Totktonada — APPROVED** [review]
I see no downsides.

**Pull request merged**